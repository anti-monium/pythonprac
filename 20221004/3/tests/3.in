[114, 13, [1, 1], 33, 'cvbn', 'hi ', 0x33], [51, [1], 'cvbn', 333, 'ewewwe']
