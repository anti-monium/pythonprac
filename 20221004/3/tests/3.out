[114, 13, [1, 1], 33, 'hi ']
